apps/website/src/server/notifications.ts

```diff
@@ -200,6 +201,24 @@ async function createDiscordNotifications({
     default:
   }
 
+  const linkify = new LinkifyIt();
+
+  const matches: LinkifyIt.Match[] | null = linkify.match(message);
+
+  let offset = 0;
+  if (matches != null) {
+    for (const match of matches) {
+      const linkStart: number = match.index + offset;
+      const linkEnd: number = match.lastIndex + offset;
+
+      const linkified = `<${match.text}>`;
+      message =
+        message.slice(0, linkStart) + linkified + message.slice(linkEnd + 1);
+
+      offset += linkified.length - (match.lastIndex - match.index + 1);
+    }
+  }
```

Comment:
As noted in the comment, I would suggest moving this into a separate helper function like `escapeLinksForDiscord(text: string)` and move it in its own util file.